qt/schedulewidget.py

```diff
@@ -205,7 +206,7 @@ def _repeated_unit_combobox(self):
 
         return combobox.BitComboBox(self, repeatedly_units)
 
-    def _slot_schedule_mode_changed(self, idx):
+    def _slot_schedule_mode_changed(self, _idx):
```

Comment:
Is there a reason we can't remove the `idx` argument altogether? I guess the calling code expects to pass an argument?

Reply:
Qt calls this slot (a signal handler) and past that argument.